src/sharing/ShareWrapper.js

```diff
@@ -170,6 +172,36 @@ class ShareWrapper extends React.Component<Props, State> {
     }
   };
 
+  deleteItem = toDelete => {
+    const files = [...this.state.files];
+    const filteredItems = files.filter(item => item.url !== toDelete.url);
+    this.setState({ files: filteredItems });
+    if (filteredItems.length === 0) {
+      showToast('Cancelled Share');
+      this.onShareCancelled();
+    }
+  };
+
+  renderItem = ({ item, index, separators }) => (
+    <ComponentWithOverlay
+      overlaySize={20}
+      overlayColor="white"
+      overlayPosition="bottom-right"
+      overlay={<IconCancel color="gray" size={20} onPress={() => this.deleteItem(item)} />}
```

Comment:
Buttons should be at least size 24. (Do we have this in, or linked from, our style guide somewhere? If not, we should.)

Reply:
changed overlaySize and `IconCancel`'s size to 24.

I couldn't find a style guide that said 24, and referred to the size from similar code in `AvatarItem` which is 20. Should that be changed as well?

Reply:
Hmm, good question.

So, actually the most important aspect of this is that the *touch target* should be at least 48px, to make it accessible for people to accurately hit:
https://material.io/design/usability/accessibility.html#layout-and-typography
(I just went and looked that up, and it reminded me that the needed touch-target size is that big.)

Looking at `AvatarItem`, the whole avatar is the touch target, and it's 50px across. (You can see this in the "Group PM" / "New group PM" screen: select some users, then touch anywhere on their avatar to remove them.) So we're covered there. That also means that the icon isn't really functioning as a button: it's more like a label on the whole avatar as a button, indicating what touching it will do.

Here, I think making the whole 200px preview a touch target for removing it would be counterintuitive. But perhaps we can make a whole 48px square in the corner around the "cancel" icon be the touch target?